### ISSUE TITLE
Default Bot parameters Usernames and Chatids should de ""

### ISSUE THREAD

**Issue by mcouto** [windkh/node-red-contrib-telegrambot#5]
**Problem:**
For telegram commands work properly, default parameters should be changed.

**Error:**
Function `telegrambot.isAuthorized()` aways returns false.

**File:**
`telegrambot/99-telegrambot.html`

_Lines 8-12:_

``` javascript
defaults: {
            botname: { value: "fooBot", required: true },
            usernames: { value: "[]", required: false },
            chatids: { value: "[]", required: false }
        },

```

_Shoud be:_

``` javascript
defaults: {
            botname: { value: "fooBot", required: true },
            usernames: { value: "", required: false },
            chatids: { value: "", required: false }
        },

```

**Description:**
Default usernames/chatids should be "" (empty string), and not empty arrays "[]"
Values may be included, without array markers ("[" and "]"). Ex: "john,peter"
Also change help in line 44 about that.

**Comment by windkh** [comment]
Thanks for reporting this. I will integrate your suggestions soon.

**Comment by MrkWkngr** [comment]
Wow. This just cost me 2hrs. Apart from that this is pretty cool.

**Comment by windkh** [comment]
Fixed in Version 1.11.0
